text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,153 @@
+
+### Tier-1 Components
+
+When discussing the components' maturity, we often debate what should be included in the evaluation criteria. To help clarify that, we introduce the concept of "Tier-1 SIGs" and "Tier-1 Components". Tier-1 Components are critical for a specific SIG: only when all Tier-1 Components are stable can a project be evaluated as stable. Example:
+
+* The "OTLP receiver" for the OpenTelemetry Collector could potentially be a "Tier-1 Component". Only after this component is declared stable may the Collector be declared stable, provided all other Tier-1 Components are also stable.
+
+Projects MUST list their Tier-1 Components in the main readme file. A Tier-1 Component MAY be hosted under the project's "contrib" repository. Not all components under the main repository (sometimes called the "core" repository) are automatically Tier-1 Components.
+
+A component can be declared "Tier-1" only if they have the following characteristics:
+
+* The component has at least the same maturity level as the SIG
+* The component has a clear code owner
+* The code owner is active within the project
+* The maintainers of the project are comfortable supporting the component should the code owner decide to step down or leave the project
+
+### Tier-1 Signals
+
+Similar to a Tier-1 Component, a Tier-1 Signal must have stable support in order to declare OpenTelemetry as stable. Declaring a signal as Tier-1 means that all projects must also see that signal as required for declaring the project stable.
+
+Not all signals will have the same requirements. For instance, Logs might not provide an instrumentation API specification.
+
+List of Tier-1 signals and their capabilities:
+
+* Logs
+  * Data model
+  * SDK specification
+  * Instrumentation SDKs
+  * Bridge API specification
+* Metrics
+  * Data model
+  * API specification
+  * Instrumentation APIs
+  * SDK specification
+  * Instrumentation SDKs
+* Traces
+  * Data model
+  * API specification
+  * Instrumentation APIs
+  * SDK specification
+  * Instrumentation SDKs
+
+The lifecycle of a signal is defined in the [Versioning and stability for OpenTelemetry clients](https://opentelemetry.io/docs/reference/specification/versioning-and-stability). While the stability and versioning of the signals may relate to this OTEP, the previously linked document specifies the details on how to reach the "stable" level mentioned in this OTEP.
+
+The "experimental" level from the signal versioning and stability document relates to the "alpha" and "beta" levels from this OTEP, "stable" and "deprecated" are equivalent to their counterparts on this document, and "removed" is non-existent here.
+
+When a new signal reaches the "stable" level, OpenTelemetry TC and GC will ultimately decide whether to include it as part of OpenTelemetry's priority components, using the process described in [Promoting SIGs and components](#promoting-sigs-and-components).
```

Comment:
```suggestion
When a new signal reaches the "stable" level, OpenTelemetry TC and GC will ultimately decide whether to include it as part of OpenTelemetry's Tier-1 components, using the process described in [Promoting SIGs and components](#promoting-sigs-and-components).
```

Reply:
I guess you found what was the term we had before tier-1 :-) Thank you for spotting this!